### DISCUSSION TITLE
Hbase region-count out of control when less than 3

### DISCUSSION THREAD

**Discussion by jodk** [JanusGraph/janusgraph#2554] [Q&A]
default min region count is 3 ,  but region-count <=  3 is out of control in storage manage

**Comment by storyteller28th** [comment]
I don't understand what you're trying to ask!
In the source code,janusgraph min region count is 3.
If you don't set the property's value,when janusgraph first run,it will create hbase's table with 9 regions

**Reply by storyteller28th** [reply]
I said something wrong，^-^,when hbase's table is created,it will create 9 family.
